examples/apps/generic_client/generic_client.py

```diff
@@ -182,6 +185,10 @@ def _parse_command_line(self, args):
             "-rs", "--requester_signature",
             help="Enable requester signature for work order requests",
             action="store_true")
+        parser.add_argument(
+            "-as", "--attestation_service",
+            help="Uri of avalon attestation service",
+            type=str)
```

Comment:
Is avs uri a mandatory parameter ?  I don't think we should make it mandatory.  It's upto the client to verify worker or not though it's recommended (eg: in case of SGX simulator we don't need verification)

Reply:
It is not mandatory parameter.